src/Detached.Mappers.EntityFramework/Conventions/EFConventions.cs

```diff
@@ -46,13 +46,32 @@ public void Apply(MapperOptions mapperOptions, ClassType typeOptions)
 
                     foreach (var inheritedType in entityType.Model.GetEntityTypes())
                     {
-                        if (inheritedType.BaseType == entityType)
```

Comment:
this is for 13 too
the comparison doesn't work because there are many levels of inheritance
needed to introduce IsBaseType to solve it

Reply:
One of the most important lines. Thanks for pointing out. 👍